src/components/pages/room.less

```diff
@@ -0,0 +1,5 @@
+.Room {}
```

Comment:
This class needs to use `.m-content` mixin 
```
.Room {
   .m-content;
}
```
so that it has the same page styling as the others.

Example:
<img width="322" alt="Screen Shot 2021-12-30 at 10 20 48" src="https://user-images.githubusercontent.com/8189923/147738558-582d9e84-ca9c-4eb1-bb2a-472d52dbf3de.png">
vs
<img width="289" alt="Screen Shot 2021-12-30 at 10 20 56" src="https://user-images.githubusercontent.com/8189923/147738573-568b2acd-3746-4d65-8bdf-fc7ad93020cc.png">

Reply:
hmm, will remember this, thanks!